pkg/roachprod/install/cluster_synced.go

```diff
@@ -2940,36 +2940,16 @@ func (c *SyncedCluster) ParallelE(
 		wg.Wait()
 	}()
 
-	var writer ui.Writer
-	out := l.Stdout
-	if options.Display == "" {
-		out = io.Discard
-	}
-
-	var ticker *time.Ticker
-	if !config.Quiet {
-		ticker = time.NewTicker(100 * time.Millisecond)
-	} else {
-		ticker = time.NewTicker(1000 * time.Millisecond)
-		fmt.Fprintf(out, "%s", options.Display)
-		if l.File != nil {
-			fmt.Fprintf(out, "\n")
-		}
+	spinner := ui.NewDefaultCountingSpinner(l, options.Display, count)
+	if options.Display != "" {
+		defer spinner.Start()()
 	}
-	defer ticker.Stop()
-
-	var spinner = []string{"|", "/", "-", "\\"}
```

Comment:
Seeing other usages of the old spinner here:

https://github.com/cockroachdb/cockroach/blob/fbf35e0d1e562dd5372c40b336ae40bad301232e/pkg/roachprod/install/cluster_synced.go#L2184

https://github.com/cockroachdb/cockroach/blob/fbf35e0d1e562dd5372c40b336ae40bad301232e/pkg/roachprod/install/cluster_synced.go#L2572

Reply:
Good catch! I'll add some commits to cover those as well.

Reply:
Ah I remember why I skipped these. They are a bit more intricate, may need to be a follow-up PR, but I should at least create an issue that these still need converting.

Reply:
Adding a new `TaskSpinner` to support `Get` / `Put`. Adds quite a few commits, but will be up today at some point.